style.css

```diff
@@ -1,3 +1,7 @@
+body {
+    background-color:aqua;

```

Comment:
@lillian-busingye kindly change the background color from `aqua` to maybe `gray` as the black text color is hard on the eyes of the clients against the `aqau` background color. WDYT?

Reply:
@lillian-busingye the code LGTM 🎯